resources/shaders/xray_composite.shader

```diff
@@ -50,10 +51,12 @@ fragment =
 
         result = layer0 * layer0.a + result * (1.0 - layer0.a);
 
-        float intersection_count = (texture2D(u_layer2, v_uvs).r * 255.0) / 5.0;
-        if(mod(intersection_count, 2.0) == 1.0)
+        float intersection_count = texture2D(u_layer2, v_uvs).r * 51; // (1 / .02) + 1 (+1 magically fixes issues with high intersection count models)
```

Comment:
Why the 0.02 then?

Reply:
0.02 is the base color which is added onto a pixel for each face behind that pixel.
It's defined in xray.shader

Reply:
About the magic formula:
I have no idea, but 255/5=51
I don't understand why the xray works the way it does, but at least now it says it's magic in a comment, rather than being undocumented.